packages/automerge-repo/src/synchronizer/DocSynchronizer.ts

```diff
@@ -173,3 +166,10 @@ export class DocSynchronizer
     })
   }
 }
+
+
+function getChangesOfMessage (decoded: DecodedSyncMessage): any  {
```

Comment:
maybe rename to something more specific: `decodeChangesToJson` but i might just reiterate these two lines if it's only ever going to be there twice